OrbitGl/App.cpp

```diff
@@ -160,6 +160,10 @@ void OrbitApp::ProcessSamplingCallStack(LinuxCallstackEvent& a_CallStack) {
 
 //-----------------------------------------------------------------------------
 void OrbitApp::ProcessHashedSamplingCallStack(CallstackEvent& a_CallStack) {
+  if (Capture::GSamplingProfiler == nullptr) {
+    ERROR("GSamlingProfiler is null, ignoring callstack event.");
```

Comment:
Typo: GSamplingProfiler in error message.

Reply:
done